packages/back-end/src/routers/demo-datasource-project/demo-datasource-project.router.ts

```diff
@@ -0,0 +1,25 @@
+import express from "express";
+import z from "zod";
+import { wrapController } from "../wrapController";
+import { validateRequestMiddleware } from "../utils/validateRequestMiddleware";
+import * as rawDemoDatasourceProjectController from "./demo-datasource-project.controller";
+
+const router = express.Router();
+
+const demoDatasourceProjectController = wrapController(
+  rawDemoDatasourceProjectController
+);
+
+router.post(
+  "/",
+  validateRequestMiddleware({
+    body: z
+      .object({
+        // TODO:
```

Comment:
This should be the expected fields for the project interface

Reply:
Hmm, I don't think it is actually (this is for the demo datasource project, not all projects). The demo datasource endpoint request should have nothing in it since we hard-code the description and just use the organization id for the project id.

Reply:
True, that makes sense. I think I was looking at the model, forgot we hardcoded these.